docs/docs/reference/release-notes/v3.3/index.md

```diff
@@ -0,0 +1,115 @@
+---
+date: "2021-04-13"
+version: "3.3.0"
+---
+
+# [v3.3](https://github.com/gatsbyjs/gatsby/compare/gatsby@3.3.0...gatsby@3.3.0) (April 2021 #1)
+
+Welcome to `gatsby@3.3.0` release (April 2021 #1)
+
+Key highlights of this release:
+
+- [Performance optimizations](#performance-optimizations) - faster JS bundling, lower peak memory usage
+- [Upgrade to the latest `sharp`](#upgrade-to-the-latest-sharp) - built-in image optimizations; and now works on M1 Macs
+- [Upgrade to the latest `remark`](#upgrade-to-the-latest-remark) - more consistency in Markdown rendering
+
+Also check out [notable bugfixes](#notable-bugfixes--improvements).
+
+**Bleeding Edge:** Want to try new features as soon as possible? Install `gatsby@next` and let us know
+if you have any [issues](https://github.com/gatsbyjs/gatsby/issues).
+
+[Previous release notes](/docs/reference/release-notes/v3.1)
```

Comment:
Any reason we skipped over 3.2 as most recent previous release notes?

Reply:
That was copy-pasta. Thanks for noticing, fixed this.